ui/tests/acceptance/secret-engine-list-view-test.js

```diff
@@ -115,26 +115,34 @@ module('Acceptance | secret-engine list view', function (hooks) {
 
   test('enterprise: cannot view list without permissions inside namespace', async function (assert) {
     this.version = 'enterprise';
-    this.backend = `bk-${this.uid}`;
     this.namespace = `ns-${this.uid}`;
+
+    const enginePath1 = `kv-${this.uid}`;
+    const userDefault = await runCmd(createTokenCmd()); // creates default user token
+    await runCmd(mountEngineCmd('kv', enginePath1));
+
     await runCmd([`write sys/namespaces/${this.namespace} -force`]);
-    await loginNs(this.namespace, ' ');
+    await loginNs(this.namespace, userDefault);
 
     await visit('/vault/secrets');
-    assert.dom(SES.secretsBackendLink('cubbyhole')).doesNotExist();
+    assert.strictEqual(currentURL(), `/vault/secrets`, 'Should be on main secret engines list page.');
 
-    await logout();
+    assert.dom(SES.secretsBackendLink(enginePath1)).doesNotExist();
   });
 
   test('enterprise: can view list with permissions inside namespace', async function (assert) {
     this.version = 'enterprise';
```

Comment:
This should probably be deleted since `enterprise:` means this test will only run on enterprise and this isn't doing anything. `this.version` doesn't seem to be associated with anything?